capstone-frontend/src/components/Navbar.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import '../App.css';
+
+class Navbar extends Component {
+  constructor(props) {
+    super(props);
+    this.toggleNavbar = this.toggleNavbar.bind(this);
+    this.toggleNavbarLibrary = this.toggleNavbarLibrary.bind(this);
+    this.state = {
+      navCollapsed: true,
+      navLibraryCollapsed: true
+    };
+  }
+
+  toggleNavbar() {
+    this.setState({
+    navCollapsed: !this.state.navCollapsed,
+      navLibraryCollapsed: true
+    });
+  }
+
+  toggleNavbarLibrary() {
+    this.setState({
+      navLibraryCollapsed: !this.state.navLibraryCollapsed,
+    });
+  }
+
+  render() {
+    const navCollapsed = this.state.navCollapsed;
+    const navClassOne = navCollapsed ? 'collapse navbar-collapse' : 'collapse navbar-collapse show';
```

Comment:
You could also use a template literal here to use the ternary for just the dynamic "show" portion of the string.

Reply:
Good idea! Cleaned it up.